cql3/restrictions/statement_restrictions.cc

```diff
@@ -210,6 +210,10 @@ static std::vector<expr::expression> extract_partition_range(
         }
 
         void operator()(bool) {}
+
+        void operator()(const unresolved_identifier&) {
+            on_internal_error(rlogger, "extract_partition_range(unresolved_identifier");
```

Comment:
Is the unmatched opening bracket intentional? It looks weird

Reply:
No, accidental, will fix before everyone's brain melts

Reply:
Fixed.